### PR TITLE
Add structural data stack support under ScanImage5

### PR DESCRIPTION
**Pull request by eywalker**
Reader4 is now compatible with structural stack (ScanImage5 stack recorded without fast Z slices).

### CODE REVIEW

File: lib/+ne7/+scanimage/Reader4.m [hunk 1]
```diff
@@ -58,12 +60,21 @@
             end
         end
         
+        function f = get.is_functional(self)
+            if self.scanimage_version == 4
```
Comment (dimitri-yatsenko) [inline lib/+ne7/+scanimage/Reader4.m:64]: so this always returns false for scanmage 4?

Comment (eywalker) [inline lib/+ne7/+scanimage/Reader4.m:64]: That was supposed to be `1` always for scanimage 4. Updated.

Basically I couldn't easily find a way to tell structural vs functional apart for scan image 4 so disabling this feature.

Comment (dimitri-yatsenko) [inline lib/+ne7/+scanimage/Reader4.m:64]: f = self.scanimage_version == 4 || self.header.hFastZ_enable;

Comment (eywalker) [inline lib/+ne7/+scanimage/Reader4.m:64]: Yeah I guess l will simplify it till I figure out the switch for version 4

**Pull request merged**